client/react/package.json

```diff
@@ -1,14 +1,18 @@
 {
-  "name": "replicache-todo-react",
+  "name": "replicache-quickstarts-todo-react",
   "private": true,
   "version": "0.1.0",
   "type": "module",
   "scripts": {
     "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
     "dev": "vite",
     "build": "npm run lint && tsc && vite build",
+    "build:server": "cd ../../server && npm run build",
     "preview": "vite preview",
-    "format": "prettier --write './src/*/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'"
+    "format": "prettier --write './src/*/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'",
+    "clean": "rm -rf ./dist && mkdir -p ./dist",
+    "prod": "npm run build:server && npm run clean && npm run build && cp -r ./dist/ ../../server/dist/ && cd ../../server && npm run prod",
+    "server": "cd ../../server && npm run dev"
```

Comment:
suggest adding a watch to react and ts-web-component packages that does

`concurrently --kill-others 'npm run dev:server' 'npm run typecheck -- --watch' 'npm run dev:react'`